### DISCUSSION TITLE
How do I set `resolve.alias` to specify only strings starting with `@/` but not `@babel`?

### DISCUSSION THREAD

**Discussion by nyngwang** [webpack/webpack#18145] [Q&A]
**update**: I found that my problem disappeared after resolving an issue related to Babel. Should not fix the following problem before any new clue is shown.

**Comment by alexander-akait** [comment]
Pleae avoid using `@`, I don't recommend it, please use any other character, for example `_`, also duplicate https://github.com/webpack/webpack/issues/18035
